templates/taxi/driver_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+
+
+{% block content %}
+	<h1>{{ object|yesno:"Update,Create" }} driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Submit" class="btn btn-primary">
+  </form>
+{% endblock %}
+
```

Comment:
looks like you have two blank lines at the end of some files. Fix that, there should be only one line